gvfs-helper.c

```diff
@@ -3600,7 +3600,7 @@ static enum gh__error_code do_sub_cmd__config(int argc, const char **argv)
 
 	if (ec == GH__ERROR_CODE__OK)
 		printf("%s\n", config_data.buf);
-	else
+	else if (ec != GH__ERROR_CODE__HTTP_404)
 		error("config: %s", status.error_message.buf);
```

Comment:
We have to add a flag to allow us to suppress 404 error message, and then pass that flag _only_ from the caller where a 404 should be handled silently.